modules/core/04-channel/keeper/events.go

```diff
@@ -83,3 +83,57 @@ func EmitTimeoutPacketEvent(ctx sdk.Context, packet exported.PacketI, channel ty
 		),
 	})
 }
+
+// EmitSendPacketEvent emits an vent with Packet data along with other packet information for relayer
```

Comment:
Missing `e` in `vent`.
And also there's no reason to make the `p` of `Packet` uppercase.